cmd/shipper-state-metrics/main.go

```diff
@@ -21,11 +24,12 @@ import (
 )
 
 var (
-	masterURL    = flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
-	kubeconfig   = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	resyncPeriod = flag.String("resync", "5m", "Informer's cache re-sync in Go's duration format.")
```

Comment:
This flag should be dropped from shipper deployment object, otherwise it will fail to start: https://github.com/bookingcom/shipper/blob/master/kubernetes/shipper.deployment.yaml#L28

Reply:
That deployment is for the `shipper` cmd. `shipper-state-metrics.deployment.yaml` doesn't pass any arguments.

Reply:
You are absolutely right.